iterations/2.md

```diff
@@ -15,13 +15,13 @@ Discuss the design of your application with your advisor. You may need to revise
 # First Feature
 
 <small>
-You may need to revise parts of the [Design](/iterations/1#design), and even parts of the [Project Proposal](/iterations/0#project-proposal) as you develop the application.
+You may need to revise parts of the [Design](/iterations/1#design), and even parts of the [Project Proposal](/iterations/0#project-proposal) as you develop the application. Remember to update the documents as required.
```

Comment:
I like this, but please cut the “remember to.” Just say “Update the documents as required.” Also, please propagate this change to the other iterations (which repeat this sentence).

Reply:
Resolved

Reply:
Please propagate this change to the other iterations (which repeat this sentence).